Makefile

```diff
@@ -15,9 +15,9 @@
 # limitations under the License
 #
 
-.PHONY: help clean clean-dist build dev test test-travis release pip-release bin-release dev-binder .binder-image
+.PHONY: help clean clean-dist build dev test test-travis release pip-release bin-release dev-binder .binder-image audit
 
-VERSION?=0.1.0.dev5
+VERSION?=0.1.0.dev5-incubating
```

Comment:
Does pip have the same notion of snapshot, where new 'deploy' overrides previous one ? If so, I would just call it 0.1.0.dev.incubating.